doc/design/cluster_status.md

```diff
@@ -0,0 +1,32 @@
+## Cluster status reporting
+
+An etcd cluster can permanently fail when all members are dead due to massive nodes failure. A user might update etcd cluster TPR with invalid input or format. The operator needs to notify users these bad events.
```

Comment:
"when all members are dead due to massive nodes failure." ->
"when all members are dead and no backup is available."